crates/wasm/src/tx.rs

```diff
@@ -103,34 +82,174 @@ pub fn build_tx(
     return serde_wasm_bindgen::to_value(&tx).unwrap();
 }
 
+/// Get transaction view, transaction perspective
+/// Arguments:
+///     full_viewing_key: `bech32 String`
+///     tx: `pbt::Transaction`
+/// Returns: `TxInfoResponse`
+#[wasm_bindgen]
+pub async fn transaction_info(full_viewing_key: &str, tx: JsValue) -> Result<JsValue, Error> {
+    let fvk = string_to_fvk(full_viewing_key)?;
```

Comment:
I'm not sure the `string_to_fvk` helper function makes sense if it's only one line. Think it can be done in-line.